po/Makevars

```diff
@@ -8,7 +8,9 @@ subdir = po
 top_builddir = ..
 
 # These options get passed to xgettext.
-XGETTEXT_OPTIONS = --keyword=_ --keyword=N_ --keyword=ERROR --keyword=PRINT
+XGETTEXT_OPTIONS = --keyword=_ --keyword=N_ --keyword=ERROR --keyword=PRINT \
+                   --keyword=SSS_TOOL_COMMAND:2 --keyword=SSS_TOOL_COMMAND_FLAGS:2 \
+                   --keyword=SSS_TOOL_DELIMITER:2
```

Comment:
Shouldn't it say SSS_TOOL_DELIMITER:1? Or perhaps even without the number.

```c
#define SSS_TOOL_DELIMITER(message) {"", _(message), 0, NULL, 0}
```

It has only one argument.

Reply:
Thanks, fixed.